examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/bin.ts

```diff
@@ -486,10 +487,11 @@ program
           process.exit(1);
         }
 
+        const token = await fetchTokenCLI(args);
         await generateApprovalPolicies({
           entity_id,
           entity_user_id,
-          token: await fetchTokenCLI(args),
+          token: token,
```

Comment:
We prefer to not include the code in the PR, which does not really change the code or improve it. It just increases the size of the PR and makes it harder to review.